### ISSUE TITLE
June 2021 Production BC127 Melody Version

### ISSUE THREAD

**Issue by biomurph** [Tympan/Tympan_Rev_E_Hardware#21]
Been poking at the new hardware and testing the BC127.
I programmed the example sketch  `RenameTympanBT` from the `feature_Teens4_SD_ble` branch of the Library
I changed line 36 to `TympanRev::E` instead of `TympanRev::D`

I'm using the Beta 9 version of Teensduino in Arduino 1.8.15
Running on MacOS Big Sur 11.4

Side note: I had to disable my virus software to run Teensyduino. It was crashing on some library example...

This is the serial port out put from the RenameTympanBT sketch. 
This module is running on Melody 6. It's likely that all 40 of this production run are Melody 6.
There definitely seem to be some basic coms differences between 6 and 7. The uniqui ID is there, but the current firmware doesn't parse it well.

From past history, I think the plan is to roll back the Melody version, since we are dependent upon the Bluetooth and all of our phone apps know is Melody 5.

One feature that we will lose is the low-latency that the versions 6 and 7 offer. 
I think it's a long-term feature add to upgrade our firmware and apps to interface with Melody 7. 

Tympan: forceBTtoDataMode: Setting PIO5 to HIGH
*** RenameTypmanBT.  Starting...
*** Switching Tympan RevD BT module into command mode...
Tympan: forceBTtoDataMode: Setting PIO5 to LOW
*** Should be in command mode.
*** Clearing buffers.  Sending carraige return.
ERROR 0x0012*** Should have gotten 'ERROR', which is fine here.
*** Restoring factory defaults...
BlueCreation Copyright 2017Melody Audio V6.1.5 HDBuild: 14969OKBlueCreation Copyright 2017Melody Audio V6.1.5 HDBuild: 14969*** Reset should be complete.  Did it give two 'OK's, an ID blurb, and then 'READY'?
*** Begin renaming process.  Getting BT name...*** Response = NAME=BC-1C1427OK
*** Processing name info...
*** Desired New BT Name = TympanE-
*** Setting New BT Name...TympanE-
OKOKNAME=TympanE-OK*** Name setting complete.  Did it return the desired name?
*** Setting GPIOCONTROL mode to OFF, which is what we need...
ERROR 0x001AOK*** Changing into transparanet data mode...
ERROR 0x0012Tympan: forceBTtoDataMode: Setting PIO5 to HIGH
*** BT Setup complete.
*** Now entering BT <-> USB Echo mode...feel free to send messages back and forth.

**Comment by chipaudette** [comment]
Yes, the short / medium plan is to ensure consistency across the fleet...by rolling back to V5.5.  This is what the App has been tested with.

Chip
